README.md

```diff
@@ -5,25 +5,12 @@
 # Postgres Connection Walkthrough
 
 This exercise is designed to get you familiar with connecting to a database, querying it and viewing that information.
-We'll be using the NPM module PG to connect our node server to a locally-hosten Postgres database.
+We'll be using the NPM module PG to connect our node server to a locally-hosted Postgres database.
 
 Inspired by [Matthew Glover](https://github.com/matthewglover) and his awesome [pg-app](https://github.com/matthewglover/pg-app/blob/master/get_pups.js)
```

Comment:
Seems odd to be linking to a file instead of the repo's main page here. Looks like @shiryz is the author of this line. Is there a good reason for it to be this way?